project_PassMeCash_Polkadot.md

```diff
@@ -0,0 +1,164 @@
+# PassMeCash
+
+- **Team Name:** CodeUp Technologies Limited
+- **Payment Address:** 0x65d68c8d14cc4acf02ec4e8d572399620a684b57 (ERC 20)
+- **[Level]:** 2
+
+
+### Overview
+
+PassMeCash is a decentralized banking system that meet the financial needs of the public through the public using crypto (DOT) & fiat. PassMeCash is a web3 financial solution that will help our target audience access their funds for payment transactions seamlessly and effortlessly through our decentralized agents.
+Unlike the traditional centralized banking system where accounts holder funds are kept in bank building and also withdrawn from same bank building; PassMeCash is a community and decentralized financial solution, where the PassMeCash users can request for their funds and it will be available to them for withdrawal via the nearest PassMeCash agent to them. 
```

Comment:
1. In my country Nigeria, people receive fiat currency from Point Of Sales (POS) operators with some minimal charges. Thus the agents facilitating these payments for PassMeCash will be with some charges from users. And this will be done via crypto to fiat, unlike the current state in Nigeria that is only fiat to fiat.
2. Yes, also we have done our survey analysis and we understand clearly the process of getting PassMeCash agents. It is also key to note that our approach will be from one city to another. PassMeCash must be fully on ground in a particular city before moving to the next. This approach is not that very expensive as the requirements to become an agent is not that difficult. Just having a smart phone, KYC, and a specific small amount of cash to dispense on a daily basis to users, as well as some other requirements qualifies one to become a PassMeCash agent.

Reply:
> 3\. they are contain in the PassMeCash Figma designs

I don't see the kind of technical details in Figma that I would like to see, such as tech stack, a list of smart contract functions, an architecture or flow diagram, etc.